pkg/library/container/container.go

```diff
@@ -65,7 +64,10 @@ func GetKubeContainersFromDevfile(workspace *dw.DevWorkspaceTemplateSpec, securi
 		if err != nil {
 			return nil, err
 		}
-		handleMountSources(k8sContainer, component.Container, workspace.Projects)
+		err = handleMountSources(k8sContainer, component.Container, workspace)
+		if err != nil {
+			return nil, err
+		}
```

Comment:
(Optional, feel free to just resolve)

```suggestion
		if err := handleMountSources(k8sContainer, component.Container, workspace); err != nil {
			return nil, err
		}
```

I always think it's a little weird that Go code goes around reassigning `err` over and over again